SofaKernel/modules/SofaBaseTopology/src/SofaBaseTopology/SparseGridTopology.cpp

```diff
@@ -19,36 +19,26 @@
 *                                                                             *
 * Contact information: contact@sofa-framework.org                             *
 ******************************************************************************/
-#include <sofa/simulation/Node.h>
 #include <SofaBaseTopology/SparseGridTopology.h>
-#include <sofa/core/visual/VisualParams.h>
+
+#include <sofa/core/behavior/MechanicalState.h>
 #include <sofa/core/topology/BaseMeshTopology.h>
 #include <sofa/core/ObjectFactory.h>
 #include <sofa/helper/io/Mesh.h>
-#include <sofa/core/loader/MeshLoader.h>
-#include <sofa/helper/system/FileSystem.h>
 #include <sofa/helper/fixed_array.h>
 #include <sofa/helper/polygon_cube_intersection/polygon_cube_intersection.h>
-#include <sofa/helper/system/FileRepository.h>
-#include <sofa/defaulttype/VecTypes.h>
+#include <sofa/core/loader/VoxelLoader.h>
```

Comment:
could you double check this one? Maybe BaseLoader?

Reply:
Already checked and it is needed:
```
void SparseGridTopology::buildAsFinest(  )
{
    VoxelLoader *loader;
    getContext()->get(loader);
    if( loader )
    {
        buildFromVoxelLoader(loader);
    }
    else
    ...
```

Reply:
beurk